server/models/User.model.ts

```diff
@@ -71,9 +71,11 @@ class User extends Model<UserAttributes, UserCreationAttributes> {
   website: string;
 
   @CreatedAt
+  @Column({ field: "created_at" })
   createdAt: Date;
```

Comment:
I'd also recommend changing this to match the expected naming convention: `created_at`